lib/src/components/modal/modal.scss

```diff
@@ -76,11 +77,11 @@ $modal-small-width: 400px !default;
   }
 }
 
-.modal-header__tabs {
-  @apply flex flex-row w-full items-center relative justify-center h-full;
+.modal-header {
+  @apply flex flex-row w-full items-center relative justify-center h-16;
 
-  .modal-header__tabs-text {
+  .modal-header__text {
     font-size: 1.125em;
     @apply font-semi-bold absolute left-0 pl-6;
   }
-}
\ No newline at end of file
+}
```

Comment:
ModalHeader will always have this style, not just when it's got tabs, so I thought I'd make it generic!